### PR TITLE
chore: release v0.6.0

### PR DESCRIPTION
**Pull request by b5**
## Change checklist

- [ ] Self-review.
- [ ] Documentation updates if relevant.
- [ ] Tests if relevant.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,34 @@
+# [v0.6.0](https://github.com/n0-computer/iroh/compare/v0.6.0-alpha.1...v0.6.0) (2023-09-25)
+
+v0.6.0 is a big step toward iroh 1.0. It introduces documents, the See our [docs pages](https://iroh.computer/docs) for a detailed look at the new iroh.
```
Comment (ramfox) [inline CHANGELOG.md:3]:
```suggestion
v0.6.0 is a big step toward iroh 1.0. It introduces documents. See our [docs pages](https://iroh.computer/docs) for a detailed look at the new iroh.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,34 @@
...
+
+## Introducing documents
+
+Documents are mutable key-value stores that authors read from, write to, and sync with, subscribing to live updates in real time. For more on documents, see the [docmuents, uh, documentation](https://iroh.computer/docs/layers/documents)
```
Comment (ramfox) [inline CHANGELOG.md:7]:
```suggestion
Documents are mutable key-value stores that authors read from, write to, and sync with, subscribing to live updates in real time. For more on documents, see the [documents, uh, documentation](https://iroh.computer/docs/layers/documents)
```

**Review by ramfox — CHANGES_REQUESTED** [review]
some spelling nits

**Comment by dignifiedquire** [comment]
merged manually, @b5 @ramfox please PR fixes directly to `main`

**Pull request merged**